### ISSUE TITLE
Error on script to install prerequisites

### ISSUE THREAD

**Issue by wimel** [dappnode/DAppNode#491]
Using the [script to install prerequisites](https://docs.dappnode.io/get-started/installation/custom-hardware/installation/script/#prerequisites)  `sudo wget -O - https://prerequisites.dappnode.io | sudo bash`  I have a warning and wireward package is not install properly.

* Steps to reproduce:

```
sudo apt update && sudo apt -y upgrade
sudo wget -O - https://prerequisites.dappnode.io | sudo bash
```

The logs show this:

```
E: Unable to locate package wireguard-dkms

 WARNING: wireguard kernel module is not installed, Wireguard DAppNode package might not work! 
```

I resolved with `sudo apt install wireguard-dkms -y`

More information about Kernel and version of Debian:

```
Distributor ID:	Debian
Description:	Debian GNU/Linux 10 (buster)
Release:	10
Codename:	buster
--
Linux DAppNode 4.19.0-21-amd64 #1 SMP Debian 4.19.249-2 (2022-06-30) x86_64 GNU/Linux
```

_I reproduce this error on two servers more and have the same problem, but I have solved it as described above, Thanks!_

┆Issue is synchronized with this [Basecamp todo](https://3.basecamp.com/5459570/buckets/29540291/todos/5380348864) by [Unito](https://www.unito.io)

**Comment by alexpeterson91** [comment]
I had the same issue trying to setup another local node last week. I had changed something with the script install last week or 2 weeks ago specifically about wireguard but it was just adding it to the packages to be installed under all scenarios and it wasn't on the prerequisites script was only a line on the install script. I failed on the perquisite script when trying to install last week too.
